rclcpp/include/rclcpp/node.hpp

```diff
@@ -127,7 +128,9 @@ class Node
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr,
-    bool ignore_local_publications = false);
+    bool ignore_local_publications = false,
+    typename rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr
+    msg_mem_strat = rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::create_default());
```

Comment:
Is there no "better" way which is compliant with uncrustify which makes the last two lines have a different indentation to show that it is a continued line?

Same below.